Keycloak/cloud-init.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+apt update -y
+apt install -y docker*
+systemctl enable --now docker.service
+
+
+# Generate certificates using Docker
+docker run -v $PWD/certs:/certs \
+  -e CA_SUBJECT="My own root CA" \
+  -e CA_EXPIRE="1825" \
+  -e SSL_EXPIRE="365" \
+  -e SSL_SUBJECT="${keycloak_server_name}" \
+  -e SSL_DNS="${keycloak_server_name}" \
+  -e SILENT="true" \
+  superseb/omgwtfssl
```

Comment:
`superseb/omgwtfssl` was updated almost 6 years ago. Can we maybe create an image using it as base image and update it for CVEs and such?

Reply:
I have updated the SSL creation process using Certbot.